vendor-bin/roave-backward-compatibility-check/composer.json

```diff
@@ -13,6 +13,9 @@
             "cweagans/composer-patches": true,
             "bamarni/composer-bin-plugin": true,
             "phpstan/extension-installer": true
+        },
+        "platform": {
+            "php": "8.3.9"
```

Comment:
personally I don't like to pin the PHP version. why is that necessary? is there something still not compatible?